### DISCUSSION TITLE
Ordination at group level, rather than row level

### DISCUSSION THREAD

**Discussion by hrlai** [JenniNiku/gllvm#191] [Q&A]
Hi there,

I have a dataset with repeated sampling at each site. For instance, my `row.eff` argument would take the formula: `~ (1 | Site) + (1 |Time)`

For a model-based ordination, the latent variables are estimated for each row by default, i.e., a site-time combination. Would it be possible to estimate species residual correlations across site and across time separately? 

I am thinking of some new argument like `lv.eff = ~ (1 | Site) + (1 |Time)`. Is this similar in philosophy to the experimental `lvCor` argument?

I would also like to hear from the crowd if this makes any sense at all. I could think of species might be less correlated in space but more correlated in time, if they partition habitat use across time so they could still be found in the same site across time. Thanks!

**Comment by BertvanderVeen** [comment]
See issue #94

**Comment by dhercar** [comment]
I have also been looking for something like that for a while. My understanding is that now you can do it with ` lvCor =~ (1 | Site) + (1 |Time) `(see [this](https://github.com/JenniNiku/gllvm/blob/master/vignettes/vignette9.Rmd) ), but I did not manage to make it work in a 4th corner model.

**Comment by BertvanderVeen** [comment]
Thanks both. You can actually do this via a random effects constrained ordination using the lv.formula argument. The below is abit of a workaround, as it is technically an unconstrained ordination, despite the fact that we are not using the `num.lv` argument. Ultimately it would be nice to do something like this via the `num.lv` argument, but this might get to what you want for the moment!

For example:

```
data(spider, package = "mvabund")
X = data.frame(group=rep(1:4, each = 7))
model<-gllvm(spider$abund, num.RR=2, X = X,lv.formula=~(1|group),family="poisson", randomB = "LV")
getLV(model)
```
